### PR TITLE
[16.0][mig] account_invoice_tax_note: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: account_invoice_tax_note/reports/report_invoice_document.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html) -->
+<odoo>
+    <template
+        id="report_invoice_document_inherit"
+        inherit_id="account.report_invoice_document"
+    >
+        <span t-field="o.fiscal_position_id.note" position="after">
```
Comment (JavierIniesta) [inline account_invoice_tax_note/reports/report_invoice_document.xml]: This line should be changed to the following:

```
<p name="note" position="after">
```

File: account_invoice_tax_note/reports/report_invoice_document.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <t
+                t-set="account_tax_groups"
+                t-value="o._get_account_tax_groups_with_notes()"
+            />
+            <t t-if="account_tax_groups">
+                <t t-foreach="account_tax_groups" t-as="account_tax_group">
+                    <p t-field="account_tax_group.report_note" />
+                </t>
+            </t>
+        </span>
```
Comment (JavierIniesta) [inline account_invoice_tax_note/reports/report_invoice_document.xml]: This line should be changed to the following:

```
</p>
```

**Review by JavierIniesta — CHANGES_REQUESTED** [review]
If there is no note set in the tax position, the notes of the tax groups are not shown.

File: account_invoice_tax_note/reports/report_invoice_document.xml (hunk 2 above)
Comment (dsolanki-initos) [reply account_invoice_tax_note/reports/report_invoice_document.xml]: Hi @JavierIniesta I've done the changes as per your feedback. Kindly review it.

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Works as expected

**Comment by JordiBForgeFlow** [comment]
@dsolanki-initos Seems that this PR has conflicts. Can you rebase?

**Comment by bosd** [comment]
ping @dsolanki-initos

**Comment by bosd** [comment]
This one can be closed now

**Pull request closed without merging**